middle_end/flambda2/to_cmm/to_cmm_env.ml

```diff
@@ -653,7 +653,9 @@ and split_in_env env res var binding =
     let env, res =
       List.fold_left
         (fun (env, res) new_binding ->
-          let flambda_var = Variable.create "to_cmm_tmp" in
+          let flambda_var =
+            Variable.create "to_cmm_tmp" Flambda_kind.value (*TODO:fix*)
```

Comment:
This kind should never be used, so any kind would work here I think.

I tried to see what it would take to propagate correct kinds here, and it would involve a lot of code everywhere in `to_cmm_env.ml`, and it didn't seem worth it. Ideally, it would be great if we had an 'invalid'/error if ever found kind for this situation, but if there is'nt one, anything should do here I'd say.